config/routes.rb

```diff
@@ -18,6 +18,7 @@
     end
     member do
       post :update_metadata
+      get '/select_thumbnail', to: 'parent_objects#select_thumbnail', as: :select_thumbnail
```

Comment:
use of restful helpers instead of hard coding routes is preferred https://guides.rubyonrails.org/routing.html#adding-member-routes

```suggestion
      get :select_thumbnail
```